youtube_dl/extractor/twitch.py

```diff
@@ -206,7 +206,8 @@ class TwitchChapterIE(TwitchItemBaseIE):
 
 class TwitchVodIE(TwitchItemBaseIE):
     IE_NAME = 'twitch:vod'
-    _VALID_URL = r'%s/[^/]+/v/(?P<id>\d+)' % TwitchBaseIE._VALID_URL_BASE
+    _VALID_URL_BASE = r'https?://(?:www\.|player\.)?twitch\.tv'
+    _VALID_URL = r'%s/(?:[^/]+/v/|\?video=v)(?P<id>\d+)' % _VALID_URL_BASE
```

Comment:
Does not match http://player.twitch.tv/?t=5m10s&video=v6528877.
Should not match https://www.twitch.tv/?video=v6528877, http://player.twitch.tv/riotgames/v/6528877?t=5m10s.